sdk/resourcemover/Azure.ResourceManager.ResourceMover/src/Customized/Models/DiskEncryptionSetResourceSettings.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using Azure.Core;
+
+namespace Azure.ResourceManager.ResourceMover.Models
+{
+    public partial class DiskEncryptionSetResourceSettings : MoverResourceSettings
+    {
+        /// <summary> Initializes a new instance of DiskEncryptionSetResourceSettings. </summary>
+        /// <param name="targetResourceName"> Gets or sets the target Resource name. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="targetResourceName"/> is null. </exception>
+        public DiskEncryptionSetResourceSettings(string targetResourceName) : base(targetResourceName)
+        {
+            Argument.AssertNotNull(targetResourceName, nameof(targetResourceName));
+            ResourceType = "Microsoft.Compute/diskEncryptionSets";
+        }
```

Comment:
let us change this ctor to make it call `this()` instead
```suggestion
        public DiskEncryptionSetResourceSettings(string targetResourceName) : this()
        {
            Argument.AssertNotNull(targetResourceName, nameof(targetResourceName));
        }
```
this is because that we would like the code to be handled in generated part as much as possible.
The generated public ctor will have some initializations which increases when the swagger grows, we will have to manually maintain this ctor to contain the same initializations as well if we do not call `this()`.

Reply:
Also we need to add the `EditorBrowsable(EditorBrowsableState.Never)` attribute to this ctor too, as well as all the other ctors we added for backward compatibility